mono-dotnet-node-xvfb/Dockerfile

```diff
@@ -14,6 +14,18 @@ RUN apt-get -q update && \
     zlib1g \
     && rm -rf /var/lib/apt/lists/* /tmp/*
 
+# Install OpenJDK-8 && Fix certificate issues
+RUN apt-get update && \
```

Comment:
Merge with the prevuous RUN command (line 5)

Reply:
done